### DISCUSSION TITLE
404 errors - bootstrap

### DISCUSSION THREAD

**Discussion by Dodgo1** [mesa/mesa#1863] [Q&A]
Hi, I am getting theese 404 errors when opening apge of Modular server in browser:

404 GET /static/external/bootstrap-slider-11.0.2/dist/css/bootstrap-slid
er.min.css (127.0.0.1) 7.43ms
404 GET /static/external/bootstrap-slider-11.0.2/dist/bootstrap-slider.m
in.js (127.0.0.1) 1.00ms

I have no idea what's causing it. I reinstalled all pacakges multiple times. 
please help, thx

**Comment by SecJiahui** [comment]
I met the same problem

**Reply by wang-boyu** [reply]
The fix should've been released - could you please try `pip install mesa-viz-tornado==0.1.3` and see if it works?

**Reply by SecJiahui** [reply]
thanks, it works for me!

**Reply by wang-boyu** [reply]
Great!
